src/main/daml/Daml/Finance/Util/Date/DayCount.daml

```diff
@@ -204,15 +217,32 @@ calcDcf30E360 toDateIsMaturityDate fromDate toDate =
     d1Adj = if d1 == monthDayCount y1 m1 then 30 else d1
     d2Adj = if d2 == 31 || not toDateIsMaturityDate && isEndOfFebruary (y2, m2, d2) then 30 else d2
   in
-    calc30360Formula (y1, m1, d1Adj) (y2, m2, d2Adj)
+    calc30FixFormula (y1, m1, d1Adj) (y2, m2, d2Adj) 360
+  where
+    (y1, m1, d1) = toGregorian fromDate
+    (y2, m2, d2) = toGregorian toDate
+
+-- | Calculate 30E3/360 day count fraction.
```

Comment:
shouldn't it be the `30E3/360`?

Reply:
Yes, I must have mixed them up. We already had E3.